### DISCUSSION TITLE
Prioritizing Outcomes

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#43] [General]
We have created a large list of outcomes. Some are familiar and some are new. There is no way to work on all of them for a first draft of WCAG 3.  We need to decide how we will prioritize the list. Below is a proposed prioritization approach. 

1. Prioritize outcomes that have quality supporting research or are foundational
- Some outcomes may be foundational/self-evident (we would need a way to determine this) and these may not have research but should still be included as self evident
- AG will work with the research questions taskforce to get a more thorough lit review of outcomes without supporting research. This list will be organized based on importance of the gap and public interest. AG will work with RQTF to coordinate research on high priority items.
2. Within the group of remaining items, prioritize outcomes that are covered in WCAG 2.2.  There will not be a perfect 1-to-1 relationship because of the changes in structure. If we want to publish something comparable to 2.2, these need to be a high priority.
3. Within the remaining set, organize by importance of the gap-filled, interest of public and difficulty (writing and meeting). 
- Pick a few items that are high public interest
- Supplement with high importance, easy wins

**Comment by u9000-Nine** [comment]
> 2. Within the group of remaining items, prioritize outcomes that are covered in WCAG 2.2. There will not be a perfect 1-to-1 relationship because of the changes in structure. If we want to publish something comparable to 2.2, these need to be a high priority.

Since WCAG&nbsp;3 is not meant to replace WCAG&nbsp;2 (yet), would it make more sense to prioritize outcomes that are *not* covered in WCAG&nbsp;2.2? That would mean that we come right out of the gate with new accessibility considerations for developers, testers, et al, instead of considerations that are already published in one form or another.

**Reply by GreggVan** [reply]
I think your list is great.   Since W3 is meant to replace W2 when it comes out -- people will be looking for whether it is a viable replacement from the start.  So include 2.x stuff as early as possible - in the new form and format - will be an important re-assurance.  But putting that second -- to our highest priority items is the right thing to do -- since as noted above -- people are interested first in what is new rather than just what is old.   Then tackle the rest as soon as we can figure each one out -- so priority would be driven by importance AND our ability to figure out how to address them.      so  +1 to your plan

**Reply by dav-idc** [reply]
I had interpreted the list posted at the top of this discussion as prioritizing a number of outcomes which aren't covered in WCAG 2.2:
> 1. Prioritize outcomes that have quality supporting research or are foundational

In my interpretation, that would include all outcomes (regardless of whether they're in WCAG 2.2) where we have at least one of these:
a) a solid supporting research
b) a 'self-evident' foundational outcome
And within that priority 1 set then, there would be some that weren't part of WCAG 2.2.

Then after the 1st priority of all the solidly-researched stuff is covered, the 2nd priority is outcomes porting over from WCAG 2.2, regardless of the current state of the research on them. I would assume that means some WCAG 2.2 pieces require further research, to meet the "quality supporting research or are foundational" threshold.

**Reply by GreggVan** [reply]
yes dav-idc   -- that is how I read it too  (if you have an OR between a and b)   And I think there would HAVE to be things in #1 that are not 2.2     

That said -- my only concern is that we not make it an "a and b" since "outcomes" need to be testable and I don't want us to spin out on important things because of testable.  I think we need to find a way to get foundational things in that may not be outcomes.  Or we fail to achieve one of our primary goals. I note that it was "or" in Rachael's post  and just not clear in yours so I think we are all on the same page.

**Reply by dav-idc** [reply]
@GreggVan good catch on the 'or' for a) or b). Yeah, that's my interpretation as well. I've updated my previous comment a bit to specify only one or the other is needed. Thanks!

**Comment by awkawk** [comment]
@rachaelbradley I agree with 1 as the top priority. For what is next I'm finding it hard to project how to organize that set without knowing what is in it. What is implied is that there are SC in WCAG 2.x that are neither research-based or foundational, which I won't disagree with, but if that is accurate do we want to commit to including those next? I'm not sure if we do...

I think that we need to be focusing on outcomes that are high-impact on end-users, and not care so much about whether it is represented in 2.x. Of course we will want to do a comparison of 2.x/3 coverage as we go...

**Comment by chaals** [comment]
(It's not obvious where the list of outcomes is, and that would be helpful in the discussion.)

If it is possible for anyone to make a proposal, and people work on proposals according to their energy levels, expertise, personal taste, etc, then there will be different priorities in the group. I the best case that means we move forward faster across a range of outcomes, and I think that is ideal. Different people focusing on different outcomes helps us test many aspects at once.

What works well for outcomes that are less critical might be different from what works for the ones that are high-impact, so seeing how to work with a range of different outcomes helps us identify any structural issues earlier.

There are several different dimensions that contribute to impact - size of user group affected, the productivity gain that a particular outcome provides, whether it's for a group we serve relatively well or relatively poorly, to suggest some. So I think it might be a useful personal rule of thumb to aim for high impact, but I don't think the group will be very productive if it spends a lot of time discussing potential impact in advance, just to decide which thing to try working on next.

I am not convinced that it is impossible to wok on everything for a first draft, although it's unlikely. A lot depends on how much gatekeeping there is.
